packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance.Client/NimbleBlazor.Tests.Acceptance.Client.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
```

Comment:
Note: With the .NET 8 updates, Blazor projects that want to use both the `InteractiveServer` and `InteractiveWebAssembly` render modes will have 2 projects. The `.Client` project has only the components that use `InteractiveWebAssembly`/`InteractiveAuto` render modes.

As mentioned in the PR description, I haven't yet made this change for the Spright Blazor acceptance tests.

Reply:
#2462 now includes supporting this for Spright. Keeping thread open in case reviewers feel it should be part of this PR (if so let me know).